core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java

```diff
@@ -1545,6 +1548,10 @@ private void updateUrlProcessor() {
     }
   }
 
+  private void updateNightMode() {
+    painter.update(getCurrentWebView(), kiwixWebView -> !(kiwixWebView.getUrl() != null && kiwixWebView.getUrl().equals(HOME_URL)), videoView);
```

Comment:
the boolean expression here can be simplified, negating the entire condition is rarely a good decision, I wouldn't be surprised if there was a quick fix